build.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
 }
```

Comment:
Can you please update version to 1.7.0

see: https://intellij-support.jetbrains.com/hc/en-us/community/posts/6826927374226-Unable-to-build-plugin-for-2022-2-after-updating-settings-gradle

Reply:
Hello there @amibiz , the latest version is actually at 1.8.1 I think, but I can bump it to 1.7.0 for sure.

Just let me know because actually I receive building errors even updating it with 1.7.x or 1.8.x version.

When I first open the ergo-keys project (cloning a new repo), the Gradle JVM uses the Liberica 1.8 Java version, and it doesn't works for me even after updating to 1.7.0.

I had to change it to use OpenJDK in my machine, once I do that, all versions including using the plugin at 1.3.0 version build without erros.

<img width="1440" alt="ergo-keys_openjdk" src="https://user-images.githubusercontent.com/2954722/187785822-c4c5a631-3b80-4969-97f2-f91d81184f6c.png">

Maybe that's because I'm on Apple Silicon and there is something wrong for this architecture on Liberica JDK.

I don't work with Java, I don't know if changing the Gradle JVM SDK to openjdk like I did is a local change in my machine or if it'll be something attached to the project.

And I don't know if you chose Liberica JDK for any particular reason.
(It was you by the way or IntelliJ just pick that randomly on my machine?)

Anyway I'll be updating it to 1.7.0 as asked.

Reply:
I ended up updating it to 1.8.1 to remove warnings.